config/tips.txt

```diff
@@ -0,0 +1,136 @@
+You can add metal into soda-can IEDs to increase the amount of shrapnel created when detonated.
+The best hiding spaces for stolen or illict items can often be in plain sight: O2 lockers, disposals, toolboxes and closets can all serve as great places to store items.
+Smugglers Satchels can be hidden underneath a floor tile, making a great small stash for supplies.
+The Nuclear Authentication disk can be tracked with the pinpointer device, making stealing it incredibly risky.
+Smart weaponry can be location tracked using the use-of-force computer software, always be careful when stealing smart weapons.
+Taking a GPS with you when going on Away Missions will increase the likelyhood of being found if deceased.
+Going into space while an EVA Ban is in effect is a good way to go missing for the rest of the round.
+The Trajan, Antonine and Hadrian can all independantly explore sectors without the ICS Nerva, just ensure to bring a canister for thrust fuel.
+There is a emergency entrance somewhat far from the Permanent Brig - if prepared well enough, you could just reach it.
+It is good to plan to get caught, especially if committing a grand-crime, as this gives you time to stash escape items.
+The Singularity and Supermatter should never touch.
+Dionae are unable to commit violent acts.
+Unathi are negatively affected by consuming food or drink with sugar.
+Lifts are great at going up and down Z-levels, but it's a good idea not to put anything below them.
+Different gas mixes will affect how the Supermatter generates heat and cools, it's always good to ask the Chief Engineer how it works.
+Hacking the access wire on a vending machine will allow you to purchase items without the needed access level.
+Chameleon projectors are most effective when projecting a small or tiny item, such as a ciggarette butt, try to match the environment you are in.
+Glass shards will cause pain and knock you down if you aren't wearing shoes.
+Certain gloves won't fit species without being manually adjusted with wirecutters.
+Use the ship's holomaps if you want to get your bearings.
+The AI cannot AI if you take it's power...
+The Clown's Noisemaker can make a variety of discreet or annoying sounds, use this to distract someone, make a panic or just have fun!
+You can disable lobby music, ship hum ambience and admin MIDIs using the preference tab.
+Using your PDA infront of someone will make a audible sound and emote, so it's unwise to use it in a dangerous situation.
+Traitors will spawn with codewords, you can use these words with potential collaborators to identify if they are syndicate agents.
+As a Team Antagonist, always work together with your fellow antagonists, selling your team out or stating who they are over comms immediatley is against the rules.
+Traitors do not have to mutally work together and could easily backstab on another.
+Changelings can spawn as jobs that might otherwise not be spawnable as traitor antagonist, such as Detectives, Second Officers, etc.
```

Comment:
```suggestion
Changelings can spawn as jobs that might otherwise not be spawn-able as a Traitor antagonist; such as Detectives, Second Officers, etc.
```